tests/e2e/corrupt_test.go

```diff
@@ -339,14 +339,14 @@ func TestCompactHashCheckDetectCorruptionInterrupt(t *testing.T) {
 
 	// Wait until the node finished compaction and the leader finished compaction hash check
 	_, err = epc.Procs[slowCompactionNodeIndex].Logs().ExpectWithContext(ctx, expect.ExpectedResponse{Value: "finished scheduled compaction"})
-	require.NoError(t, err, "can't get log indicating finished scheduled compaction")
+	require.NoErrorf(t, err, "can't get log indicating finished scheduled compaction")
 
 	leaderIndex := epc.WaitLeader(t)
 	_, err = epc.Procs[leaderIndex].Logs().ExpectWithContext(ctx, expect.ExpectedResponse{Value: "finished compaction hash check"})
-	require.NoError(t, err, "can't get log indicating finished compaction hash check")
+	require.NoErrorf(t, err, "can't get log indicating finished compaction hash check")
```

Comment:
Question: there is no variable-length variable in this case, shouldn't `require.NoError` be better than `require.NoErrorf`?

Reply:
Note this is a generic question.

Reply:
We should always use `*f` functions when `require-f-funcs: true` no matter there is variable-length variables or not?